examples/docker_provider/main.tf

```diff
@@ -0,0 +1,23 @@
+provider "docker" {
+  host = "${var.docker_host}"
+}
+
+data "docker_registry_image" "chef_client" {
+  name = "errygg/chef-client"
```

Comment:
Well, spoke too soon. SSH is not enabled on the base ubuntu image. Looking into how to enable that now.